### ISSUE TITLE
When titles get incorrectly identified and are manually corrected, Jellyfin overwrites the user-made changes.

### ISSUE THREAD

**Issue by GNY-001F2** [jellyfin/jellyfin-plugin-anime#90]
So for example if I have a few series from Space Battleship Yamato and the server was unable to match the right series with each other, I manually use the "identify" setting and supply the correct AniDB or Anilist ID to fix it.

However within 24 hours my metadata updates get reversed.

It is tedious to manually identify AND lock each series after correcting the metadata. Especially when you have to do it to hundreds of series. The web interface isn't exactly the most efficient for this job either.

Another problem is that often the name supplied from AniDB/AniList isn't what I want the series to be named in my database. Yamato is again a good example of this because the name Star Blazers gets added.

After I manually change it back to Space Battleship Yamato, again it gets changed to Star Blazers after some time.

All of this is, really, really, annoying and some settings tools to put this behaviour under control would be a massive quality of life update.

**Comment by orezraey** [comment]
Yes, I have this frustration too. I don't like the standard names that are identified (in English), but I like the name of the animes in romaji.

**Comment by oddstr13** [comment]
I don't remember which one of the providers does this, but one of them are returning trash search results rather than no hits, and somehow overwriting other providers.

**Comment by GNY-001F2** [comment]
@oddstr13 Hmm that would definitely explain things. It would be helpful to know which provider was doing it so I could try removing it from my provider list for the Anime folder.

**Comment by oddstr13** [comment]
@GNY-001F2 for that, just try disabling one at a time, until the undesirable behavior stops – then report back here :wink:

**Comment by GNY-001F2** [comment]
@oddstr13 As much as that would be nice, it took me a week to make my libraries useable on Jellyfin, given the sheer size of my media content. I don't want to go through that trauma ever again 🤕
